sagemaker-pipelines/tabular/local-mode/sagemaker-pipelines-local-mode.ipynb

```diff
@@ -0,0 +1,1132 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    framework_version = "0.23-1"
</pre><p>Is it possible to use the newest 1.0-1 SKLearn version which launched in May?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Done